arch/arm/boot/dts/overlays/panel-dsi-generic-overlay.dts

```diff
@@ -0,0 +1,113 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+	compatible = "brcm,bcm2835";
+
+	dsi_frag: fragment@0 {
+		target = <&dsi1>;
+		__overlay__{
+			status = "okay";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			port {
+				dsi_out:endpoint {
+					remote-endpoint = <&panel_dsi_port>;
+				};
+			};
+			panel: panel-dsi-generic@0 {
+				// See panel-dsi.yaml binding
+				compatible = "panel-dsi";
```

Comment:
heh right, yea I'm was sure if it would be sensible to have dummy string there but yea for the sake of consistency it would probably be better to have a dummy string to the generic overlay